testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/weld/extension/BeforeShutdownJNDILookupTestCase.java

```diff
@@ -51,7 +53,11 @@
                 .create(WebArchive.class, DEPLOYMENT)
                 .addClasses(BeforeShutdownJNDILookupTestCase.class, BeforeShutdownExtension.class)
                 .add(EmptyAsset.INSTANCE, ArchivePaths.create("WEB-INF/beans.xml"))
-                .add(new StringAsset(BeforeShutdownExtension.class.getName()), "META-INF/services/" + Extension.class.getName());
+                .add(new StringAsset(BeforeShutdownExtension.class.getName()), "META-INF/services/" + Extension.class.getName())
+                .addAsManifestResource(PermissionUtils.createPermissionsXmlAsset(
+                        new FilePermission(TEST_PATH.getParent().toString(), "read, write"),
+                        new FilePermission(TEST_PATH.toString(), "read, write, delete")
```

Comment:
This doesn't matter, but 'delete' shouldn't be needed as the test driver, not the deployment, does the delete.